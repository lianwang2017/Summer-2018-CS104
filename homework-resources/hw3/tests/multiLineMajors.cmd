BUAD
EE
FNDR
CANST
CSCI
CECS