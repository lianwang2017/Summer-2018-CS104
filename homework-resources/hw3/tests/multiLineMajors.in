Max Nikias, BUAD
Max Nikias, EE
Max Nikias, FNDR
Steven B. Sample, BUAD
Steven B. Sample, EE
David Pritchard, CANST
David Pritchard, CSCI
Mark W. Redekopp, CECS
Mark W. Redekopp, EE