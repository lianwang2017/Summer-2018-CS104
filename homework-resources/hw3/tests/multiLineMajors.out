BUAD 
Max Nikias
Steven B. Sample

EE 
Mark W. Redekopp
Max Nikias
Steven B. Sample

FNDR 
Max Nikias

CANST 
David Pritchard

CSCI 
David Pritchard

CECS 
Mark W. Redekopp

